final_project/Tests/CreateAndDeleteTests.cs

```diff
@@ -0,0 +1,85 @@
+using Microsoft.Playwright;
+using final_project.Pages;
+
+namespace final_project.Tests
+{
+    [TestFixture]
+    public class CreateAndDeleteTests
+    {
+        public IPage page { get; private set; }
+        private IBrowser browser;
+        public IBrowserContext context;
+
+        public string email;
+
+        public string password;
+
+        [OneTimeSetUp]
+        public async Task OneTimeSetUp()
+        {
+            var playwrightDriver = await Playwright.CreateAsync();
+            browser = await playwrightDriver.Chromium.LaunchAsync(new BrowserTypeLaunchOptions
+            {
+                Headless = false
+            });
+            context = await browser.NewContextAsync(new BrowserNewContextOptions
+            {
+                ViewportSize = new ViewportSize
+                {
+                    Width = 1920,
+                    Height = 1080
+                }
+            });
+            page = await context.NewPageAsync();
+        }
+
+        [Test]
+        public async Task CreateAccountTest()
+        {
+            var homePage = new HomePage(page);
+            var createAccountBody = context.APIRequest.CreateFormData();
+            email = "testemailfor126@gmail.com";
```

Comment:
it is better not to do like this.
If you have some constants then just do 
 public string email =  "testemailfor126@gmail.com";
at the Class level, but not to declare undefined "public string email;" and then give it a single value in single place. In this case you can face many errors when you use "email" but it was not initialized yet.

Reply:
Done